core/admin/admin-notices.php

```diff
@@ -500,16 +500,28 @@ public function print_admin_notice( array $options ) {
 		$default_options = [
 			'title' => '',
 			'description' => '',
+			'classes' => [ 'notice' ],
```

Comment:
It's better to let the user to only provide the type of message instead of classes, then you take the type and convert it to a class.

Reply:
Some notices use other classes besides `notice-*`. See base-app.php line 581